### PR TITLE
adding coq-ext-lib 0.10.2

### PR DESCRIPTION
**Pull request by gmalecha**

### CODE REVIEW

**Comment by gmalecha** [comment]
@yazko

File: released/packages/coq-ext-lib/coq-ext-lib.0.10.2/opam [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+authors: ["Gregory Malecha"]
+license: "BSD"
+build: [
+  [make "-j%{jobs}%"]
+]
+install: [
+  [make "install"]
+]
+remove: [
+  ["rm" "-R" "%{lib}%/coq/user-contrib/ExtLib"]
```
Comment (gares) [inline released/packages/coq-ext-lib/coq-ext-lib.0.10.2/opam]: This stanza can go, opam 2 takes care of it

File: released/packages/coq-ext-lib/coq-ext-lib.0.10.2/opam [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+]
+depends: [
+  "ocaml"
+  "coq" {>= "8.8" & < "8.10~"}
+]
+synopsis:
+  "A basic library of basic Coq datatypes, definitions, theorems, and"
+description: "tactics meant to extend the standard library."
+flags: light-uninstall
```
Comment (gares) [inline released/packages/coq-ext-lib/coq-ext-lib.0.10.2/opam]: This flag as well

**Comment by gmalecha** [comment]
Thanks @gares ! I've fixed the files and a bunch of other files to try to avoid the problem in the future (since I just copy a file and modify it).

**Comment by gmalecha** [comment]
What is the timeline for merging this?

**Comment by palmskog** [comment]
Some CI job failed due to timeout after 3+ hours, but I guess it's fine to merge since the other job succeeded. For a faster turnaround time, it's always good to submit fewer packages/changes.

**Comment by gmalecha** [comment]
Thanks. I will do that in the future

**Pull request merged**